provider/datagen/src/transform/cldr/datetime/neo.rs

```diff
@@ -433,13 +433,13 @@ fn months_convert(
                 if index == 0 {
                     panic!("CLDR month indices cannot be zero");
                 }
-                *map.get(index - 1)
-                    .expect("Found out of bounds month index for calendar")
+
+                index - 1
             };
 
-            symbols.insert(code.into(), &**v);
+            symbols[index] = &**v;
```

Comment:
This doesn't work; please fix. You are currently generating this JSON. Note that CLDR-JSON month numbering for Hebrew is a complete mess.

```json
  "LeapLinear": [
    "Tishri",
    "Heshvan",
    "Kislev",
    "Tevet",
    "Shevat",
    "Adar I",
    "Adar",
    "Nisan",
    "Iyar",
    "Sivan",
    "Tamuz",
    "Av",
    "Elul",
    "",
    "",
    "",
    "",
    "",
    "Adar II",
    "",
    "",
    "",
    "",
    ""
  ]
```

Reply:
Note that the old logic uses this table to map from CLDR-JSON Hebrew month codes to Temporal month codes:

```rust
    static HEBREW_MONTH_CODES: &[TinyStr4] = &[
        tinystr!(4, "M01"),
        tinystr!(4, "M02"),
        tinystr!(4, "M03"),
        tinystr!(4, "M04"),
        tinystr!(4, "M05"),
        tinystr!(4, "M05L"),
        tinystr!(4, "M06"),
        tinystr!(4, "M07"),
        tinystr!(4, "M08"),
        tinystr!(4, "M09"),
        tinystr!(4, "M10"),
        tinystr!(4, "M11"),
        tinystr!(4, "M12"),
        // M06L is handled separately in MonthSymbols code
    ];
```